lib/clientv2/bgppeer_e2e_test.go

```diff
@@ -34,11 +34,11 @@ import (
 	"github.com/projectcalico/libcalico-go/lib/watch"
 )
 
-// Perform CRUD operations on Global and Node-specific BGP Peer Resources.
 var _ = testutils.E2eDatastoreDescribe("BGPPeer tests", testutils.DatastoreAll, func(config apiconfig.CalicoAPIConfig) {
 
-	name1 := "bgpnode-1"
-	name2 := "bgpnode-2"
+	ctx := context.Background()
+	name1 := "policy-1"
```

Comment:
why is this `policy-1`? this is bgppeer :)

Reply:
oops - must have been editing the wrong file.  ta.